spec/graphql/field_spec.rb

```diff
@@ -7,4 +7,48 @@
     end
     assert_equal(DairyProductUnion, field.type)
   end
+
+
+  describe '.property ' do
+    let(:field) do
+      GraphQL::Field.define do
+        # satisfies 'can define by config' below
+        property :internal_prop
+      end
+    end
+
+    it 'can define by config' do
+      assert_equal(field.property, :internal_prop)
+    end
+
+    it 'has nil property if not defined' do
+      no_prop_field = GraphQL::Field.define { }
+      assert_equal(no_prop_field.property, nil)
+    end
+
+    describe 'default resolver' do
```

Comment:
This test is making sure that if you change `property` after initialization, `resolve` still works?